.github/workflows/run-e2e-tests.yaml

```diff
@@ -75,50 +81,24 @@ jobs:
           token: ${{ env.TOKEN }}
           custom_version_json: ${{ needs.init-github-context.outputs.e2e-test-version-ref }}
 
-  attach-landing-zone-to-bee-workflow:
-    runs-on: ubuntu-latest
-    needs: [ init-github-context, create-bee-workflow, params-gen ]
-    steps:
-      - name: Get actions
-        uses: actions/checkout@v4
-
-      - name: dispatch to terra-github-workflows
-        id: FirstAttemptLandingZone
-        continue-on-error: true
-        uses: ./.github/actions/landing-zone-attach
-        with:
-          run_name: "attach-billing-project-to-landing-zone-${{ env.RUN_NAME_SUFFIX }}"
-          bee_name: ${{ env.BEE_NAME }}
-          token: '${{ env.TOKEN }}'
-          project_name: ${{ needs.params-gen.outputs.project-name }}
-
-      - name: retry dispatch to terra-github-workflows in case of failure
-        if: steps.FirstAttemptLandingZone.outcome == 'failure'
-        uses: ./.github/actions/landing-zone-attach
-        with:
-          run_name: "attach-billing-project-to-landing-zone-retry-${{ env.RUN_NAME_SUFFIX }}"
-          bee_name: ${{ env.BEE_NAME }}
-          token: '${{ env.TOKEN }}'
-          project_name: ${{ needs.params-gen.outputs.project-name }}
-
   run-e2e-test-job:
     needs:
-      - attach-landing-zone-to-bee-workflow
+      - create-bee-workflow
       - params-gen
     permissions:
       contents: read
       id-token: write
-    uses: broadinstitute/dsp-reusable-workflows/.github/workflows/run_tsps_e2e_tests.yaml@main
+    uses: broadinstitute/dsp-reusable-workflows/.github/workflows/run_tsps_e2e_tests.yaml@TSPS-280_teaspoons_e2e_gcp
     with:
       billing-project-name: '${{ needs.params-gen.outputs.project-name }}'
       bee-name: '${{ needs.params-gen.outputs.bee-name }}'
       e2e-test-version-ref: '${{ github.event.inputs.e2e-test-version-ref }}'
+      wdl-method-version: '${{ github.event.inputs.wdl-method-version }}'
 
   destroy-bee-workflow:
     runs-on: ubuntu-latest
     needs:
       - run-e2e-test-job
-      - attach-landing-zone-to-bee-workflow
       - init-github-context
```

Comment:
you can probably get rid of this dependency too since `run-e2e-test-job` depends on it. Probably was done by me for some dumb reason

Reply:
good catch, thanks!